<launch>
    <arg name="robot_type" default="sirius_mini" doc="Robot type: [sirius_mini,sirius_belt]"/>
		<param name="legged_robot_description" command="cat $(find mini_tail_description)/urdf/sirius_mini.urdf"/>

		<node name="generate_lcm_types" pkg="legged_sirius_hw" type="make_types.sh" output="screen"/>

		<rosparam file="$(find legged_sirius_hw)/config/$(arg robot_type).yaml" command="load"/>

		<param name="robot_type" value="$(arg robot_type)"/>
		<node name="legged_sirius_arm_hw" pkg="legged_sirius_hw" type="legged_sirius_arm_hw" respawn="false"
						clear_params="true" output="screen"/>

</launch>
